### DISCUSSION TITLE
Testing examples

### DISCUSSION THREAD

**Discussion by AvraamMavridis** [TanStack/query#402] [General]
Hi Tanner, 

Are there examples about testing? e.g. I have a component with 2 `useMutation` and I will like to write some tests related to success/error. How would you mock the `useMutation` for that? (I am using jest but any example would work)

**Comment by wolverineks** [comment]
have a look at this, is this a possibility?
https://www.polvara.me/posts/how-to-test-asynchronous-methods/
